.kube/app/Dockerfile

```diff
@@ -36,9 +37,29 @@ RUN apt-get install -y \
 RUN docker-php-ext-configure gd --with-freetype --with-jpeg
 RUN docker-php-ext-install -j$(nproc) gd pdo_mysql mbstring exif pcntl bcmath bz2 zip soap intl opcache
 
-RUN apt-get install -y libmagickwand-dev --no-install-recommends
+RUN apt-get install -y \
+    libmagickwand-dev \
+    libmagickcore-dev \
+    --no-install-recommends
 
-RUN pecl install imagick
+# START BUILDING IMAGICK FROM SOURCE - this can be removed once pecl fixes issues installing with php8.4
```

Comment:
Can you please leave a link to https://github.com/accessibility-exchange/platform/issues/2425 in the comments. It'll just make it easier to understand when scanning the code later.

Reply:
added